bindings/r/inc/RExports.h

```diff
@@ -48,7 +64,7 @@
 //Some useful typedefs
 typedef std::vector<TString> TVectorString;
 
-#include<RcppCommon.h>
+#
```

Comment:
This seems wrong

Reply:
Thanks for spotting this. It is fixed now